### PR TITLE
FizzBuzz - 26 March - v1

### PR DESCRIPTION
**Pull request by AntonKrychevskyy**

### CODE REVIEW

**Comment by taha-programmer** [comment]
Hi AntonKrychevskyy,

It looks like you've done this assignment before Khalils's had introduced the Grading checklist for the assignments, that's why some of the items in the checklist are not covered in your PR. (though your solution still works fine).

A few notes here:
Try to make "it/test" blocks contain the specific example being tested, and it's preferred to have 1 expect statement in each "it/test" statement, this way it'll help you trace failed tests.

After you see 3 "it" statements testing the same responsibility/behaviour, like "Giving Fizz for multiples of 3"), you can refactor those tests and combine them into a "describe" block and nest the "it" statement inside it, or use it.each()

So, in summary, use describe blocks to define the general behaviour/functionality, this is the abstract part, doesn't contain specific examples, and inside it nest all relevant tests with it() statements where each it() contains 1 expect().

Here is a review that Khalil's done recently:
https://www.loom.com/share/cc20d756e80a4b5ba0034766d7800caf
